src/messages/en.json

```diff
@@ -23,6 +23,20 @@
         "allEntriesLabel": "Row",
         "contains": "contains"
     },
+    "DetailsPageControls": {
+        "download": "Download",
+        "embed": "Embed",
+        "fullscreen": "Fullscreen"
+    },
+    "EmbedModal": {
+        "dashboardBody": "This dashboard can be embedded using the following code:",
+        "dashboardTitle": "Embed dashboard",
+        "embedHeight": "Height",
+        "embedWidth": "Width",
+        "keepSearchParams": "Preserve search parameters",
+        "resourceBody": "The visualization for this resource can be embedded using the following code.",
```

Comment:
I think you forgot something 😅

Reply:
Ups, only changed it for the dashboard but not for resource 😅